docs/en/table_design/hybrid_table.md

```diff
@@ -8,10 +8,21 @@ As an OLAP database, StarRocks originally stores data in the columnar storage, w
 
 ## Comparisons between columnar storage and hybrid row-column storage
 
-| **Storage format**         | **Storage method**                                           | **Scenarios**                                                |
-| -------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| Hybrid row-column storage | Data is stored in both row-by-row and column-by-column fashions. Simply put, a table that uses hybrid row-column storage contains an additional, hidden binary-type column `__row`. When data is written to the table, all values from the value columns of a row are encoded and written to the `__row` column (as shown below). As the data is stored in both row-by-row and column-by-column fashions, additional storage costs are incurred. ![img](../assets/table_design/hybrid_table.png) | <ul><li>Suitable for scenarios such as primary key-based point queries and partial column updates, because this storage method can help greatly reduce random IO and read-write amplification in these scenarios.</li><ul><li>Point queries, which are primary key-based simple queries that scan and return small amounts of data.</li><li>Queries against most or all of the fields from tables that consist of a small number of fields.</li><li>Partial column updates.</li><li>Prepared statements can be run on tables that use hybrid row-column storage, which can enhance query performance by saving the overhead of parsing SQL statements, and also prevent SQL injection attacks.</li></ul><li>Also suitable for complex data analysis.</li></ul>|
-| Column-oriented            | Data is stored in a column-by-column fashion ![img](../assets/table_design/columnar_table.png) | <ul><li>Complex or ad-hoc queries on massive data. </li><li>Tables (such as wide tables) consist of many fields. Queries involve only a few columns. </li></ul>|
+**Hybrid row-column storage**
+
+- Storage method：Data is stored in both row-by-row and column-by-column fashions. Simply put, a table that uses hybrid row-column storage contains an additional, hidden binary-type column `__row`. When data is written to the table, all values from the value columns of a row are encoded and written to the `__row` column (as shown below). As the data is stored in both row-by-row and column-by-column fashions, additional storage costs are incurred.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/55affb4d-4598-488e-bbcb-1006db69f401)
Is it "each involved row"? Because it cannot be every row or any random row, which are logically unacceptable. (Attend to the cn doc too.)